primitives/consensus/common/src/import_queue/basic_queue.rs

```diff
@@ -63,9 +84,22 @@ impl<B: BlockT, Transaction: Send + 'static> BasicQueue<B, Transaction> {
 			finality_proof_import,
 		);
 
+		let guard = Arc::new((Mutex::new(0usize), Condvar::new()));
+		let guard_start = guard.clone();
+		let guard_end = guard.clone();
+
 		let mut pool = futures::executor::ThreadPool::builder()
 			.name_prefix("import-queue-worker-")
-			.pool_size(1)
+			.pool_size(2)
```

Comment:
Why do we increase the size?

Reply:
Isn't there just one worker anyway?

Reply:
Was testing if thread tracking code is working properly. Eventually there should be more workers, but this is out of scope of this PR.